wordpress-mobile/WordPress-iOS
# Quick Start: Alert Appearance

Refs #9447

This improves the appearance of the quick start alerts:

![quick start alert styling](https://user-images.githubusercontent.com/517257/44074712-9edbc290-9f4e-11e8-8235-1ad947ff3144.gif)

### Tech discussion
Two changes of note:
- Method `createHighlightMessage(base: String, highlight: String, icon: UIImage) -> NSAttributedString` for creating the strings with highlights icons + words
- Enum case `quickStart(NSAttributedString?)` for passing the attributed string created by the above to the Notice

To test:
- View the Quick Start checklist for any wpcom site
- Tap an incompleted checklist item
- Ensure the notice/alert that appears at the bottom of the screen resembles the mockup
